packages/react/src/utils/assert.spec.ts

```diff
@@ -1,16 +1,17 @@
+/* eslint-disable vitest/expect-expect */
 import { assert } from '.'
 
-function get(type: 'Helper' | 'Core') {
+function get(type: 'foo' | 'bar') {
   switch (type) {
-    case 'Helper':
+    case 'foo':
       return {
-        type: 'Helper',
-        library: 'Suspensive',
+        foo1: 'foo1',
+        foo2: 'foo2',
       } as const
-    case 'Core':
+    case 'bar':
       return {
-        type: 'Core',
-        library: 'React',
+        bar1: 'bar1',
+        bar2: 'bar2',
```

Comment:
It's better that the test code doesn't contain the domain context as much as possible.(for dummy object)

This is because it can cause confusion to people who read test code. Originally, there was a problem that the person reading the test code read the unnecessary domain contents. I replaced it with foo and bar so that I can understand it without thinking.

Reply:
So much better!! Thanks to make our library avoid confusing to contributors